src/map/script.cpp

```diff
@@ -25812,6 +25812,30 @@ BUILDIN_FUNC(randomoptgroup)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC( open_quest_ui ){
+	struct map_session_data* sd;
+
+	if (!script_charid2sd(3, sd))
+		return SCRIPT_CMD_FAILURE;
+
+	int quest_id;
+
+	if (!script_hasdata(st, 2))
+		quest_id = 0;
+	else {
+		int i;
+		quest_id = script_getnum(st, 2);
+
+		ARR_FIND(0, sd->avail_quests, i, sd->quest_log[i].quest_id == quest_id);
+		if (i == sd->avail_quests)
+			ShowWarning("buildin_open_quest_ui: Character %d doesn't have quest %d.\n", sd->status.char_id, quest_id);
```

Comment:
```suggestion
		quest_id = script_getnum(st, 2);

		if( quest_id != 0 ){
			int i;
			ARR_FIND(0, sd->avail_quests, i, sd->quest_log[i].quest_id == quest_id);
			if (i == sd->avail_quests)
				ShowWarning("buildin_open_quest_ui: Character %d doesn't have quest %d.\n", sd->status.char_id, quest_id);
		}
```

Reply:
If we do this, now we need to change the doc again ;)